MOD_NAME/MOD_NAME.csproj

```diff
@@ -0,0 +1,40 @@
+<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<AssemblyName>MOD_NAME</AssemblyName>
+		<TargetFramework>net48</TargetFramework>
+		<LangVersion>latest</LangVersion>
+		<Nullable>enable</Nullable>
+		<WarningsAsErrors>CS8600;CS8602;CS8603</WarningsAsErrors>
+	</PropertyGroup>
+
+	<!-- Derail Valley -->
+	<ItemGroup>
+		<Reference Include="Assembly-CSharp"/>
+	</ItemGroup>
+
+	<!-- Unity -->
+	<ItemGroup>
+		<Reference Include="UnityEngine"/>
+		<Reference Include="UnityEngine.CoreModule"/>
+	</ItemGroup>
+
+	<!-- Mod Loader -->
+	<ItemGroup>
+		<PackageReference Include="UnityModManager" Version="0.27.2"/>
```

Comment:
This is cool! I guess this lets us avoid having to include it in the `Directory.Build.targets` file?

Is 0.27.3 available? Not sure if it changes much in this context, but my understanding is that that's the version we need for everything to work the way we asked it to be changed to.

Reply:
> This is cool! I guess this lets us avoid having to include it in the Directory.Build.targets file?

Yup!

> Is 0.27.3 available?

0.27.2 is the latest on NuGet. The only difference between the two is in Derail Valley's configuration.